spellcheck.pl

```diff
@@ -239,13 +260,17 @@ sub spellcheck_add_word
 Irssi::command_bind('spellcheck_add', 'spellcheck_add_word');
 
 Irssi::settings_add_bool('spellcheck', 'spellcheck_enabled', 1);
+Irssi::settings_add_bool('spellcheck', 'spellcheck_print_suggestions', 1);
 Irssi::settings_add_str( 'spellcheck', 'spellcheck_default_language', 'en_US');
 Irssi::settings_add_str( 'spellcheck', 'spellcheck_languages', '');
 Irssi::settings_add_str( 'spellcheck', 'spellcheck_word_color', '%R');
 Irssi::settings_add_str( 'spellcheck', 'spellcheck_window_name', '');
 Irssi::settings_add_str( 'spellcheck', 'spellcheck_window_height', 10);
+Irssi::settings_add_str( 'spellcheck', 'spellcheck_word_input_color', '%U');
```

Comment:
Any particular reason the default is underline, rather than red? I think on the screenshot red looked pretty good.

Reply:
just what I liked